tests/setup_local_db/test_data_results_and_cdm.sql

```diff
@@ -27,37 +27,70 @@ values
+    (4,2000000324,1993,5,22,'1993-05-22 00:00:00',NULL,8516,0,NULL,NULL,NULL,'d90c07cc-e303-298a-6d28-fbac7ff3f282','F',0,'black',0,'',0),
+    (5,2000000324,1953,11,11,'1953-11-11 00:00:00',NULL,8515,0,NULL,NULL,NULL,'e6b6627f-4e38-dfc8-078c-11406151c521','F',0,'asian',0,'',0),
+    (6,2000000324,1958,11,11,'1958-11-11 00:00:00',NULL,8515,0,NULL,NULL,NULL,'e6b6627f-4e38-dfc8-078c-11406151c522','F',0,'asian',0,'',0)
 ;
 
 insert into omop.observation
 (observation_id,person_id,observation_concept_id,observation_date,observation_datetime,observation_type_concept_id,value_as_number,value_as_string,value_as_concept_id,qualifier_concept_id,unit_concept_id,provider_id,visit_occurrence_id,visit_detail_id,observation_source_value,observation_source_concept_id,unit_source_value,qualifier_source_value,observation_event_id,obs_event_field_concept_id,value_as_datetime)
 values
-    (1,1,2000000324,'2019-03-29','2019-03-29 00:00:00',38000276,NULL,'F',0,0,0,NULL,26,0,'43878008',0,NULL,NULL,NULL,0,NULL),
-	(22,1,2000000324,'2013-04-15','2013-04-15 00:00:00',38000276,NULL,'F',0,0,0,NULL,9,0,'302870006',0,NULL,NULL,NULL,0,NULL),
-	(23,2,2000000324,'2014-02-05','2014-02-05 00:00:00',38000276,NULL,'A value with , comma!',0,0,0,NULL,52,0,'278860009',0,NULL,NULL,NULL,0,NULL),
-	(35,2,0,'2017-06-13','2017-06-13 00:00:00',38000276,NULL,NULL,0,0,0,NULL,60,0,'444814009',0,NULL,NULL,NULL,0,NULL),
-	(47,3,2000000324,'1993-10-24','1993-10-24 00:00:00',38000276,NULL,'M',0,0,0,NULL,81,0,'713197008',0,NULL,NULL,NULL,0,NULL),
-	(48,3,0,'1967-12-02','1967-12-02 00:00:00',38000276,NULL,NULL,0,0,0,NULL,114,0,'53741008',0,NULL,NULL,NULL,0,NULL),
-	(57,4,2000000324,'2019-02-16','2019-02-16 00:00:00',38000276,NULL,'F',0,0,0,NULL,162,0,'198992004',0,NULL,NULL,NULL,0,NULL),
-	(58,4,0,'2012-06-06','2012-06-06 00:00:00',38000276,NULL,NULL,0,0,0,NULL,170,0,'403191005',0,NULL,NULL,NULL,0,NULL),
-	(64,5,2000000324,'1993-11-17','1993-11-17 00:00:00',38000276,NULL,NULL,0,0,0,NULL,179,0,'162864005',0,NULL,NULL,NULL,0,NULL),
-	(65,5,0,'2014-01-31','2014-01-31 00:00:00',38000276,NULL,NULL,0,0,0,NULL,197,0,'278860009',0,NULL,NULL,NULL,0,NULL),
-	(66,5,0,'2020-03-16','2020-03-16 00:00:00',38000276,NULL,NULL,0,0,0,NULL,184,0,'84229001',0,NULL,NULL,NULL,0,NULL),
+    (nextval('observation_id_seq'), 1,2000000324,'2019-03-29','2019-03-29 00:00:00',38000276,NULL,'F',0,0,0,NULL,26,0,'43878008',0,NULL,NULL,NULL,0,NULL),
+	(nextval('observation_id_seq'), 1,2000000324,'2013-04-15','2013-04-15 00:00:00',38000276,NULL,'F',0,0,0,NULL,9,0,'302870006',0,NULL,NULL,NULL,0,NULL),
+	(nextval('observation_id_seq'), 2,2000000324,'2014-02-05','2014-02-05 00:00:00',38000276,NULL,'A value with , comma!',0,0,0,NULL,52,0,'278860009',0,NULL,NULL,NULL,0,NULL),
+	(nextval('observation_id_seq'), 2,0,'2017-06-13','2017-06-13 00:00:00',38000276,NULL,NULL,0,0,0,NULL,60,0,'444814009',0,NULL,NULL,NULL,0,NULL),
```

Comment:
curious why don't we set observation_concept_id for some values?

Reply:
good question @UchicagoZchen138 ! I added an extra comment [here](https://github.com/uc-cdis/cohort-middleware/pull/27/commits/57586798454e853a81897c04a2cd20eae349d682) and [here](https://github.com/uc-cdis/cohort-middleware/pull/27/commits/33b26edcc086b58bcc14916e25d5a78b5f11faa8) that should help clarify this a bit.

Reply:
Ok, added test here now: https://github.com/uc-cdis/cohort-middleware/pull/27/commits/969d4766fcba6d7faeb4d87ff18563574a2d45eb